src/helm/benchmark/presentation/summarize.py

```diff
@@ -158,6 +151,38 @@ def get_method_display_name(model_display_name: Optional[str], info: Dict[str, A
     return (model_display_name or "???") + (f" [{dict_to_str(info)}]" if len(info) > 0 else "")
 
 
+def compute_row_win_rates(table: Table) -> List[Optional[float]]:
+    """
+    Computes the average win rate of each row across columns. That is, for a given row, if we pick a random column and
+    another random row, what is the probability that is row has a better value.
```

Comment:
I think this sentence is complicated to parse without variable names:

For a given row r1, if we pick another row r2 and column c1 uniformly at random, what is the probability the value in r1c1 is better than r2c1?